### DISCUSSION TITLE
I cannot use Black hole to Sample with Maschine MK3 from YouTube

### DISCUSSION THREAD

**Discussion by NITROBEATZ** [ExistentialAudio/BlackHole#817] [General]
### Please Confirm

- [X] I have read the **[FAQ](https://github.com/ExistentialAudio/BlackHole#faq) and [Wiki](https://github.com/ExistentialAudio/BlackHole/wiki)** where most common issues can be resolved
- [X] I have searched **[Discussions](https://github.com/ExistentialAudio/BlackHole/discussions)** to see if the same question has already been asked
- [X] This is a **bug** and not a question about audio routing or configuration, which should be posted in [Discussions](https://github.com/ExistentialAudio/BlackHole/discussions)

### macOS Version

macOS 13 Ventura

### BlackHole Build(s) Affected

- [X] 2 channel
- [X] 16 channel
- [ ] 64 channel
- [ ] other/custom build

### Describe the bug

I believe that I am setting up black hole correctly in both the system settings, midi set up and the Maschine app but for some reason when I am trying to sample the sound isn't getting to the Maschine, Can you please give me some help on this. I am using Maschine MK3, Sonoma and M2 Mac, Thanks . 

### Reproduction Steps

1.Set up the aggregate device and the multi output device 
2.I set up the Maschine set ups 
3.
etc...

### Expected Behavior

It should sample on Maschine but its not 

### Screenshots

<img width="1920" alt="Screenshot 2024-11-10 at 5 59 09 PM" src="https://github.com/user-attachments/assets/2fc29589-5236-4865-b148-2cf8295f6d9a">
<img width="1920" alt="Screenshot 2024-11-10 at 6 09 27 PM" src="https://github.com/user-attachments/assets/7cea8e46-6e13-4a9e-8dea-657b462cf8dc">
<img width="1920" alt="Screenshot 2024-11-10 at 6 09 32 PM" src="https://github.com/user-attachments/assets/885fabbc-cab2-415c-be98-72508609e337">

**Comment by devinroth** [comment]
This is not a bug. This is a question about audio routing.

I will convert this to a discussion.

Post a screenshot of your multi output.

Devin

**Reply by NITROBEATZ** [reply]
<img width="1920" alt="Screenshot 2024-11-12 at 9 17 05 PM" src="https://github.com/user-attachments/assets/6983af40-532e-468c-ac7d-945fe0c1a6c9">
Does this look like the routing is wrong?
